src/main/java/com/softserve/academy/spaced/repetition/controller/DeckController.java

```diff
@@ -111,9 +111,13 @@ public DeckPublicDTO getDeckById(@PathVariable Long deckId) {
         return buildDtoListForCollection(cards, CardPublicDTO.class, linkTo(methodOn(DeckController.class)
                 .getCardsByCourseAndDeck(categoryId, courseId, deckId)).withSelfRel());
     }
+    /*  TODO method addDeckToCategory had same PostMapping value as addDeckForUser()
+     *  temporarily changed categories -> categories1 in value so other
+     *  method can work
+     */
 
     @Auditable(action = AuditingAction.CREATE_DECK_IN_CATEGORY)
-    @PostMapping(value = "/api/categories/{category_id}/decks")
+    @PostMapping(value = "/api/categories1/{category_id}/decks")
```

Comment:
why did you add "1" to this url?

Reply:
It is a hotfix to a problem where two controllers had same Postmapping value
method addDeckToCategory had same PostMapping value as addDeckForUser()
temporarily changed categories -> categories1 in value so other
method can work